### PR TITLE
Custom searchbar design - FO

### PR DESCRIPTION
**Pull request by YeLnatSs**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | dev
| Description?  | New design for search bar in header.
| Type?         | improvement
| Category?     | FO
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | [#13732 ](https://github.com/PrestaShop/PrestaShop/issues/13732)
| How to test?  | Please indicate how to best verify that this PR is correct.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13058)

### CODE REVIEW

**Comment by samuel-pires** [comment]
#11

File: ps_searchbar.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
+.search-widget {
+	display: inline-block;
```
Comment (PierreRambaud) [inline ps_searchbar.css]: use spaces instead of tabs.

File: ps_searchbar.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+.search-widget form {
+	position: relative;
+}
+
+.search-widget form input[type=text] {
+	border: none;
+	padding: 10px;
+	min-width: 255px;
+	color: #7A7A7A;
```
Comment (PierreRambaud) [inline ps_searchbar.css]:
```suggestion
	color: $gray;
```

File: ps_searchbar.css [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+	border: 1px solid rgba(0, 0, 0, 0.25);
+}
+
+.search-widget form input[type=text]:focus {
+	outline: 3px #2fb5d2 solid;
+	color: $gray-darker;
+	background: white;
+}
+.search-widget form input[type=text]:focus  + button .search {
+	color: #2fb5d2;
```
Comment (PierreRambaud) [inline ps_searchbar.css]:
```suggestion
	color: $brand-primary;
```

File: ps_searchbar.css [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+.search-widget form input[type=text] {
+	border: none;
+	padding: 10px;
+	min-width: 255px;
+	color: #7A7A7A;
+	border: 1px solid rgba(0, 0, 0, 0.25);
+}
+
+.search-widget form input[type=text]:focus {
+	outline: 3px #2fb5d2 solid;
```
Comment (PierreRambaud) [inline ps_searchbar.css]:
```suggestion
	outline: 3px $brand-primary solid;
```

**Comment by atomiix** [comment]
I just did a new PR for the new search bar design (#12)

**Pull request closed without merging**